### PR TITLE
Updated port expander device driver to add label and tag for pins

### PR DESCRIPTION
**Pull request by Aditya-TY**
Updated port expander device driver to add label and tag for pins
ticket : https://trackonomy.atlassian.net/browse/FW-1606

### CODE REVIEW

**Review by aravindhulipalla — APPROVED** [review]
Looks good. @abhishek-pcsharma needs your review.

File: arch/arm64/boot/dts/freescale/imx8mp-edm-g-wb.dts [hunk 1]
```diff
@@ -300,6 +303,11 @@
 		pinctrl-names = "default";
 		#gpio-cells = <2>;
 		status = "okay";
+		/* below tags for gpio pins are updated to sapphire 2.3.4.2 */
```
Comment (abhishek-pcsharma) [inline arch/arm64/boot/dts/freescale/imx8mp-edm-g-wb.dts:306]: below tags for gpio pins are updated *with respect* to sapphire 2.3.4.2 *schematic*

File: arch/arm64/boot/dts/freescale/imx8mp-edm-g-wb.dts [hunk 2]
```diff
@@ -208,19 +208,22 @@
 		/*pinctrl-0 = <&pinctrl_pi4io16_A>;*/
 		#gpio-cells = <2>;
 		status = "okay";
+		/* below tags for gpio pins are updated to sapphire 2.3.4.2 */
```
Comment (abhishek-pcsharma) [inline arch/arm64/boot/dts/freescale/imx8mp-edm-g-wb.dts:211]: below tags for gpio pins are updated *with respect* to sapphire 2.3.4.2 *schematic*

**Review by abhishek-pcsharma — APPROVED** [review]
LGTM.

File: arch/arm64/boot/dts/freescale/imx8mp-edm-g-wb.dts (hunk 2 above)
Comment (Aditya-TY) [reply arch/arm64/boot/dts/freescale/imx8mp-edm-g-wb.dts:211]: will update in future

**Pull request merged**